### DISCUSSION TITLE
Using SuperCollider with Flok

### DISCUSSION THREAD

**Discussion by SCFan32** [munshkr/flok#289] [General]
Hi, I've been wanting to try out SuperCollider on Flok, however I don't know how the REPL configuration works, so I can't produce any sound. I've tried running the flok-repl on my Macs terminal and in the safari JS-Terminal, both with no success. Do I need any other Programs installed on my mac, and where do I run the code?
Im sorry for the inconvenience, I don't know anything other than SC so for this kind of stuff I'm pretty helpless.

Thanks for your help!

**Comment by SCFan32** [comment]
Nevermind I just figured out that I needed NodeJS, everything works perfectly now.
In case anyone has the same problem here is a link to install NodeJS: https://nodejs.org/en/download/prebuilt-installer

**Reply by munshkr** [reply]
Hi! Yes, for everything you need Node installed (`npx`, `npm`, etc. are Node commands). I just noticed it's not mentioned anywhere in the README, sorry! There's a lot to improve to the documentation...
